packages/chat-component-bindings/src/errorBarSelector.ts

```diff
@@ -40,9 +40,14 @@ export const errorBarSelector = createSelector([getLatestErrors], (latestErrors)
   if (!specificSendMessageErrorSeen && latestErrors['ChatThreadClient.sendMessage'] !== undefined) {
     activeErrors.push('sendMessageGeneric');
   }
+
+  // We only return the first few errors to avoid filling up the UI with too many `MessageBar`s.
+  activeErrors.splice(maxErrorCount);
```

Comment:
Can you mention this as a remark in the function comment, or maybe update the main line to
`Select three latest active errors from the state for the `ErrorBar` component`